modules/adagioBidAdapter.md

```diff
@@ -38,6 +47,10 @@ Connects to Adagio demand source to fetch bids.
             category: 'sport', // Recommended. Category of the content displayed in the page.
             subcategory: 'handball', // Optional. Subcategory of the content displayed in the page.
             postBid: false // Optional. Use it in case of Post-bid integration only.
```

Comment:
missing a `,` after `postBid: false` - discovered while copy/pasting to test the adapter.